spec/unit/event_storage_spec.rb

```diff
@@ -189,8 +190,10 @@ module Backend
           context 'when a ping was executed previously' do
             context 'and ping TTL is expired' do
               before do
-                stub_const("ThreeScale::Backend::EventStorage::PING_TTL", 0)
```

Comment:
@unleashed This is a bit unexpected, but as I mention in the commit description, the `expire` command accepts 0 as the TTL, but `set` with TTL does not.

Reply:
:+1: